local-volume/provisioner/pkg/discovery/discovery_test.go

```diff
@@ -394,22 +395,44 @@ func verifyPVsNotInCache(t *testing.T, test *testConfig) {
 	}
 }
 
+func TestRoundDownSize(t *testing.T) {
```

Comment:
Would this be better in util_test?

Reply:
I added the file util_test, and added these test cases.